config/index.js

```diff
@@ -762,6 +768,12 @@ var conf = convict({
       format: 'duration',
       env: 'SECONDARY_EMAIL_MIN_UNVERIFIED_ACCOUNT_TIME'
     }
+  },
+  signinCodeSize: {
```

Comment:
It's more of a mouthful but I'm replacing `signinCodesBaseUri` with `installFirefoxWithSigninCodeBaseUri`, to better advertise what it's more closely related to.